sys/net/crosslayer/ng_netif/ng_netif_addr_to_str.c

```diff
@@ -28,6 +29,8 @@ char *ng_netif_addr_to_str(char *out, size_t out_len, uint8_t *addr,
         return NULL;
     }
 
+    memset(out, 0, out_len);
```

Comment:
Change it, your use-case is valid in that regard. My solution is just generating less code ;)